contracts/UFragmentsPolicy.sol

```diff
@@ -67,6 +67,12 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
+    // The time of day in seconds (0 to 86400) around which rebase in active
```

Comment:
// The beginning of rebase time window within minRebaseTimeIntervalSec cyclic period.
// If minRebaseTimeIntervalSec = 24 hours, it's time of the day from midnight in seconds at which rebase window begins.